portfolio/src/main/java/com/google/sps/servlets/UploadUrlServlet.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+
+@WebServlet("blobstore-upload-url")
+public class UploadUrlServlet extends HttpServlet {
+
+    private static final long serialVersionUID = 1L;
```

Comment:
Indeed, I don't use it but VS Code triggers a warning when I don't add it! Should I remove it anyways ?

Reply:
what warning does it give you?

Reply:
This one: "The serializable class UploadUrlServlet does not declare a static final serialVersionUID field of type long"

Reply:
According to this article: https://www.javaworld.com/article/2077391/into-the-mist-of-serialization-myths.html

It seems like we can safely leave this out.

Perhaps you can configure VSCode to ignore this particular warning?

Otherwise, it's fine to leave it in. I'll let you decide.